dotcom-rendering/src/web/lib/decideFormat.ts

```diff
@@ -0,0 +1,9 @@
+import { decideDesign } from './decideDesign';
+import { decideDisplay } from './decideDisplay';
+import { decideTheme } from './decideTheme';
+
+export const decideFormat = (format: Partial<CAPIFormat>): ArticleFormat => ({
+	display: decideDisplay(format),
+	theme: decideTheme(format),
+	design: decideDesign(format),
+});
```

Comment:
A new method that replaces a common pattern. It’s also more resilient, as it has defaults values if the `CAPIFormat` is partial (any key missing).

Reply:
Should we allow CAPIFormat to be incomplete? That feels wrong to me. If we're missing a property there that sounds like something has gone very wrong and we probably want to throw rather than default (defaulting is likely to result in us rendering the _wrong_ thing - like, calling a sport piece news, which is bad)

Reply:
This can be addressed in another PR. This type is only a better reflection of what these methods accept. `decideDisplay`, `decideTheme` and `decideDesign` all have a `default:` in their switch. If we threw, that would be a change in the current behaviour!

In decideTrail, we did this the following:

```ts
const format: ArticleFormat = trail.format
		? {
				display: decideDisplay(trail.format),
				theme: decideTheme(trail.format),
				design: decideDesign(trail.format),
		  }
		: {
				display: ArticleDisplay.Standard,
				theme: ArticlePillar.News,
				design: ArticleDesign.Standard,
		  };
```